packages/markers/src/browser/problem/problem-tree-model.ts

```diff
@@ -65,6 +65,11 @@ export class ProblemTree extends MarkerTree<Diagnostic> {
         if (columnNumber !== 0) {
             return columnNumber;
         }
+        // Sort by owner in alphabetical order.
```

Comment:
minor, but I think we should document this in the method's header.

Reply:
```ts
/**
 * Sort markers based on the following rules:
 * - Markers are fist sorted by `severity`.
 * - Markers are then sorted by `line number` if applicable.
 * - Markers are then sorted by `column number` if applicable.
 * - Markers are then finally sorted by `owner` if applicable.
 * @param a the first marker for comparison.
 * @param b the second marker for comparison.
 */
protected sortMarkers(a: MarkerNode, b: MarkerNode): number {
```